client/my-sites/importer/importing-pane.jsx

```diff
@@ -199,7 +207,7 @@ class ImportingPane extends React.PureComponent {
 
 	render() {
 		const {
-			importerStatus: { importerId, errorData = {}, customData },
+			importerStatus: { importerId, /*errorData = {},*/ customData },
```

Comment:
Should we move this and line 224 to a TODO if we're going to put them back later?

Reply:
Thanks! Added a TODO further down and removed it from here.